ca/proto/ca.proto

```diff
@@ -66,6 +67,7 @@ message CRLMetadata {
   int64 issuerNameID = 1;
   int64 thisUpdateNS = 2; // Unix timestamp (nanoseconds), also used for CRLNumber.
   int64 shardIdx = 3;
+  google.protobuf.Timestamp thisUpdate = 4;
```

Comment:
Note that you can put numbered fields in a protobuf in any order. So if you want, you can put this right next to `thisUpdateNS` so that the fields retain their original (semi-logical?) order.